### PR TITLE
US95607-add new ifrau plugin for syncing css variable

### PR DESCRIPTION
**Pull request by edwardg8**

### CODE REVIEW

**Comment by edwardg8** [comment]
Working on fixing the test.

File: src/host.js [hunk 1]
```diff
@@ -52,7 +53,7 @@ function Host(elementProvider, src, options) {
 	if (options.syncFont) {
 		this.use(syncFont);
 	}
-
```
Comment (dlockhart) [inline src/host.js]: Can we make this an option that's `false` by default? Just worried that lots of other consumers who don't know anything about syncing CSS variables will upgrade and suddenly have some code running.

Comment (edwardg8) [reply src/host.js]: yea

**Comment by edwardg8** [comment]
I think we'll need unit test, adding one soon

**Pull request merged**